pandas/core/frame.py

```diff
@@ -1387,7 +1387,8 @@ def to_html(self, buf=None, columns=None, col_space=None, colSpace=None,
                                            escape=escape,
                                            max_rows=max_rows,
                                            max_cols=max_cols,
-                                           show_dimensions=show_dimensions)
+                                           show_dimensions=show_dimensions,
+                                           notebook=notebook)
```

Comment:
here pass directly to ``.to_html()`` (and add in the doc-string). That way this is not relevant to ``DataFrameFormatter``, just to the ``HTMLFormatter`` (which is created in the ``.to_html`` call)